spec/spec_helper.rb

```diff
@@ -7,6 +7,7 @@
 end
 
 require 'active_support/core_ext/object/blank'
+require 'active_support'
```

Comment:
This fixes an error with ActiveSupport missing a file https://github.com/rails/rails/issues/45505

Another choice would have been to switch the specific spec to use `rails_helper` but doing it this way means we can run the unit specs for this class without having to load all of Rails